test/test_da.py

```diff
@@ -0,0 +1,70 @@
+"""Tests for module da on Domain Adaptation """
+
+# Author: Remi Flamary <remi.flamary@unice.fr>
+#
+# License: MIT License
+
+import numpy as np
+import ot
+
+
+def test_otda():
+
+    n_samples = 150  # nb samples
+    np.random.seed(0)
```

Comment:
RandomState

the get_data_classif function should take the rng in param and use it instead of np.random.randn

see the check_random_state function in sklearn